project/Build.scala

```diff
@@ -64,7 +64,7 @@ object Scrooge extends Build {
     version := libVersion,
     organization := "com.twitter",
     crossScalaVersions := Seq("2.10.5" , "2.11.7"),
-    scalaVersion := "2.10.5",
+    scalaVersion := "2.11.7",
```

Comment:
If I dont do this, scrooge jars generated are for 2.10 (scrooge-runtime_2.10). I will look to see if there is a way to set it somewhere else.